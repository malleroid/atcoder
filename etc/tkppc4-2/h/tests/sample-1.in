2 1
42 8
