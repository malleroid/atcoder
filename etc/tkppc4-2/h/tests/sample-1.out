1.190476190476
